app/views/components/gds/gds_main_template.scala.html

```diff
@@ -41,8 +41,6 @@
 
 @scripts = {
   <script src='@routes.Assets.versioned("lib/govuk-frontend/govuk/all.js")'></script>
-  <script src='@routes.Assets.versioned("javascripts/google.tag.manager.min.js")'> </script>
```

Comment:
Is google tags definitely not required then?  What does Emily think?

Reply:
I'm not saying is not required. We don't have this script and this causes errors in console.

Reply:
After consideration I agree with Steve. It might be, that this line was added so that we only need to add the google script. But better safe than sorry.

Reply:
Yeah, I'll talk to Emily about it